src/api/app/policies/staging/workflow_policy.rb

```diff
@@ -1,6 +1,10 @@
 class Staging::WorkflowPolicy < ApplicationPolicy
-  def initialize(user, record)
-    super(user, record, user_optional: true)
+  def initialize(user, record, opts = {})
+    super(user, record, opts.reverse_merge(ensure_logged_in: true))
+  end
+
+  def new?
+    true
```

Comment:
If the user isn't authorized to create a staging workflow, then we shouldn't authorize them to instantiate a new staging workflow. This line would then be:

```suggestion
    create?
```